hoodie-hadoop-mr/src/test/resources/sample1.avro

```diff
@@ -1,17 +1,23 @@
 {
```

Comment:
Lets not bring in any new test schemas unless absolutely required and also lets have all test schemas accessible from SchemaTestUtil for reuse. You can add a evolved schema if you need in SchemaTestUtil like in https://github.com/uber/hoodie/pull/181/files#diff-70c2afdf2469948f545a2aefcb2d3350R108